### PR TITLE
Ratest 66: Wrote attachments feature gherkin scenarios

### PR DESCRIPTION
**Pull request by insookwa**
Wrote attachments feature gherkin scenarios in attachment.feature

### CODE REVIEW

**Comment by sherrif10** [comment]
Thanks @insookwa  for working on this, Ensure to always include ticket link , this is used to track the changes in future Feel free to follow this https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: src/test/resources/features/allergies.feature [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Feature: allergies feature
+
+  Background:
```
Comment (kaweesi) [inline src/test/resources/features/allergies.feature:3]: please exclude this file from this PR

File: src/test/resources/features/attachments.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Feature: Adding attachments
+
```
Comment (kaweesi) [inline src/test/resources/features/attachments.feature:2]: check my reviews at https://github.com/openmrs/openmrs-contrib-qaframework/pull/3/files and work on this similarly

**Review by kaweesi — COMMENTED** [review]
thanks @insookwa please check my feedback

**Comment by achilep** [comment]
cc: @insookwa

**Comment by sherrif10** [comment]
Hello @insookwa , Hope you saw @kaweesi  comments above , Are you available to continue with this Pr

File: src/test/resources/features/attachments.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Feature: Adding attachments
```
Comment (sherrif10) [inline src/test/resources/features/attachments.feature:1]: @insookwa  lets try to handle each feature file with its corresponding steps and pages in separate pr, personally i think that will make reviewing much easier , cc @kaweesi  whats your opinion on this

**Comment by kaweesi** [comment]
@insookwa are you still interested in completing this PR? please checkout and address the suggested reviews

**Comment by kaweesi** [comment]
https://github.com/openmrs/openmrs-contrib-qaframework/pull/16

**Pull request closed without merging**